src/standard/configure.html

```diff
@@ -121,6 +121,9 @@
             value = value.call(this, this._config);
           }
           config[i] = value;
+        } else if (this[i] !== undefined) {
```

Comment:
Seems like the check `if` and the `else if` should be switched here. If there's an existing value, we should take that and not the default value.